README.md

```diff
@@ -0,0 +1,175 @@
+# Transmogrifier
+The Transmogrifier is a Data Processing Pipeline, which allows for conformation of external data to standardized schemae. It can take data from multiple sources, process it through a series of dataset-specific and broad-level filters, then output the transformed data to any number of sinks (individual or grouped datasets).
+
+The filters can be highly customized to fit any given data standard or requirement and the sinks can be configured to output to a file, upload to a database, or write to a URL. 
+
+
+# Purpose
+Transmogrifier is a tool that automates the process of data extraction, transformation, and loading. The Transmogrifier supports extracting data from diverse sources, allows for multiple transformations, and facilitates loading the transformed data to various target destinations.
+
+# Manifest
+The manifest is the JSON file that defines the standardized schemae, data sources (datasets), filters, and sink(s). At the top level, it is an array of objects. Each object contains the following properties:
+- a schema
+- a top-level array of filters
+- a top-level array of sinks (optional)
+- an array of entries
+Within each entry, each object contains the following properties:
+- a source
+- an array of filters
+- an array of sinks (optional)
+
+## Features
+The Transmogrifier pipeline is outlined as the following: 
++schema
++filters (broad-level) 
++sinks
++entries
```

Comment:
Not sure where to add this but here makes the most sense to me, up to you where you actually put it. Anyways, I think that we should clarify between builtin functions such as "file_write" and "url_read" and external functions (URLs) otherwise a user may get the two confused. Maybe include something in brackets like (not supported by dart) for the external functions.
I know the dart thing is mentioned below but putting it up here as well makes sense to me.